### ISSUE TITLE
export 'observe' from '/observers' ?

### ISSUE THREAD

**Issue by D10221** [thebuilder/react-intersection-observer#402]
Hi,

First , thank you !.

I think the "observe" implementation is the hidden gem here :)
Would you consider exporting  it ?

Thanks again.

**Comment by thebuilder** [comment]
It is powerful. 
I think my main concern is that it exposes the "internals" of the library. Suddenly any changes to `observer` could be considered breaking changes.

**Comment by D10221** [comment]
As far as you keep the signature :)
``` 
observe(element: Element,  callback: ObserverInstanceCallback,  options: IntersectionObserverInit = {}) => void
```
To be honest, my initial thought was that it should be on its own lib/namespace. 
it's only dependency is 'IntersectionObserver'

**Comment by ValeryLosik** [comment]
Hi, I also want to thank you for this great project. Currently, we have started using `IntersectionObserver` in our application and we were searching for good react solution for it. But we also need more low-level API for `IntersectionObserver`, (e.g. when we work with `refs` and  we don't want to run whole reconciliation cycle) and `observe` fits ideally.

**Comment by thebuilder** [comment]
Alright, good to know. Would need to be detailed in the readme file if added, as a low level option.

**Comment by thebuilder** [comment]
Closed by #473
